pkg/scheduler/exportimport.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduler
+
+import (
+	"context"
+	"encoding/gob"
+	"errors"
+	"fmt"
+	"os"
+
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/dapr/cli/pkg/print"
+	"github.com/dapr/cli/pkg/scheduler/stored"
+)
+
+type ExportImportOptions struct {
+	SchedulerNamespace string
+	KubernetesMode     bool
+	TargetFile         string
+}
+
+type ExportFile struct {
+	Jobs     map[string][]byte
+	Counters map[string][]byte
+}
+
+func Export(ctx context.Context, opts ExportImportOptions) error {
+	if _, err := os.Stat(opts.TargetFile); !errors.Is(err, os.ErrNotExist) {
+		if err == nil {
+			return fmt.Errorf("file '%s' already exists", opts.TargetFile)
+		}
+		return err
+	}
```

Comment:
In my experience most CLI don't check for this and just overwrite the file if it exists, should we do that?